pkg/util/log/log_test.go

```diff
@@ -53,16 +72,33 @@ func TestBasicLogging(t *testing.T) {
 	Err := Error
 	Crt := Critical
 
-	Trace("%s", "bar")
-	Debug("%s", "bar")
-	Info("%s", "bar")
-	Wn("%s", "bar")
-	Err("%s", "bar")
-	Crt("%s", "bar")
+	Trace("bar")
+	Debug("bar")
+	Info("bar")
+	Wn("bar")
+	Err("bar")
+	Crt("bar")
```

Comment:
This was wrong (it was passing a format string to the non-format version). I fixed it here since I was modifying the test anyway.